testlapack/dgecon.go

```diff
@@ -44,6 +44,20 @@ func DgeconTest(t *testing.T, impl Dgeconer) {
 			condOne: 0.024740155174938,
 			condInf: 0.012034465570035,
 		},
+		/*
+			// Dgecon does not match Dpocon for this case. https://github.com/xianyi/OpenBLAS/issues/664.
+			{
```

Comment:
Additionally, is there a reason these don't get the same near miss treatment as the randomised tests?